README.md

```diff
@@ -303,9 +371,13 @@ async function createLocalSigner() {
   };
 }
 
+// read the asset
+const buffer = await readFile('to-be-signed.jpg');
+// asset mimetype must match the asset type ebing read
+const asset: Asset = { buffer, mimeType: 'image/jpeg' };
+
+// create a signing function
 async function sign(asset, manifest) {
```

Comment:
asset/buffer moved out since they'll be passed in as params to the sign function